2022/wot-cg-2022.html

```diff
@@ -0,0 +1,297 @@
+    </p>
+    <h2 id="deliverables">
+      Deliverables
+    </h2>
+    <h3 id="specifications">
+      Specifications
+    </h3>
+    <p>
+      No Specifications will be produced under the current charter
+    </p>
+    <h3 id="non-normative-reports">
+      Non-Normative Reports
+    </h3>
+    <p>
+      The group may produce other Community Group Reports within the scope of this charter 
+      but that are not Specifications, for instance use cases, requirements, implementation experience reports or white papers.
+    </p>
+    <h3 id="test-suites">
+      Test Suites and Other Software
+    </h3>
+    <p>
+      There are no plans to create a test suite.
+    </p>
+    <h2 id="liaisons">
+      Dependencies or Liaisons
+    </h2>
+    <p>
+      The Web of Things Community Group manages the dependencies and liaisons separately in other documents listed below:
+      <ul>
+        <li> <a href="https://github.com/w3c/wot-cg/blob/main/dependencies.md">Dependencies</a></li>
+        <li> <a href="https://github.com/w3c/wot-cg/blob/main/liaisons.md">Liaisons</a></li> 
+      </ul>
+      
+      Any changes to these documents need to be either a Pull Request reviewed by the chairs or agreed upon in a public meeting.
+    </p>
+    <h2 id="process">
+      Community and Business Group Process
+    </h2>
+    <p>
+      The group operates under the <a href=
+      "https://www.w3.org/community/about/agreements/">Community and Business
+      Group Process</a>. Terms in this Charter that conflict with those of the
+      Community and Business Group Process are void.
+    </p>
+    <p>
+      As with other Community Groups, W3C seeks organizational licensing
+      commitments under the <a href=
+      'http://www.w3.org/community/about/agreements/cla/'>W3C Community
+      Contributor License Agreement (CLA)</a>. When people request to
+      participate without representing their organization's legal interests,
+      W3C will in general approve those requests for this group with the
+      following understanding: W3C will seek and expect an organizational
+      commitment under the CLA starting with the individual's first request to
+      make a contribution to a group <a href="#deliverables">Deliverable</a>.
+      The section on <a href="#contrib">Contribution Mechanics</a> describes
+      how W3C expects to monitor these contribution requests.
+    </p>
+
+    <p>
+      The <a href="https://www.w3.org/Consortium/cepc/">W3C Code of
+      Ethics and Professional Conduct</a> applies to participation in
+      this group.
+    </p>
+
+    <h2 id="worklimit">
+      Work Limited to Charter Scope
+    </h2>
+    <p>
+      The group will not publish Specifications on topics other than those
+      listed under <a href="#specifications">Specifications</a> above. See
+      below for <a href="#charter-change">how to modify the charter</a>.
```

Comment:
This change seems reasonable to me (since the Specifications section is empty) but let's check with Kaz.
```suggestion
      The group will not publish on topics other than those
      listed under <a href="#deliverables">Deliverables</a> above. See
      below for <a href="#charter-change">how to modify the charter</a>.
```